proto/src/determined/api/v1/api.proto

```diff
@@ -142,6 +142,11 @@ service Determined {
         option (google.api.http) = {get: "/api/v1/experiments"};
         option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {tags: "Experiments"};
     }
+    // Get a list of experiment labels
+    rpc GetExperimentLabels(GetExperimentLabelsRequest) returns (GetExperimentLabelsResponse) {
+        option (google.api.http) = {get: "/api/v1/experiment/labels"};
```

Comment:
blocking: `experiments`?

Reply:
I actually thought singular sounded more correct there, but I'm willing to be outvoted on matters of style :)

Reply:
hmm I don't have a strong preference I was thinking since it's a namespace we haven't used and it's not exactly clear what the openAPI standard suggests maybe we avoid adding `/experiment/x` :thinking: 

side question: if it was `GET:experiments/labels` would the router correctly distinguish it with `GET:experiments/:id` it should prioritize the static route

Reply:
lmk if/when you get a chance to give the reordering a try.

Reply:
Yeah so no matter where I put the definition of this new call relative to other uses of /experiments, it seems to always expect an integer instead of /labels because of our use of /experiments/{id}.I get the same result whether GetExperimentLabels is moved before the first of /experiments, just after the first use of experiments, or after all uses of /experiments/{id}:

{"error":{"code":3,"reason":"InvalidArgument","error":"type mismatch, parameter: experiment_id, error: strconv.ParseInt: parsing \"labels\": invalid syntax"}}